pr_agent/git_providers/bitbucket_provider.py

```diff
@@ -10,12 +10,12 @@
 from ..config_loader import get_settings
 from ..log import get_logger
 from .git_provider import FilePatchInfo, GitProvider
+import ast
 
 
 class BitbucketProvider(GitProvider):
     def __init__(
-        self, pr_url: Optional[str] = None, incremental: Optional[bool] = False
-    ):
+        self, pr_url: Optional[str] = None, incremental: Optional[bool] = False):
         s = requests.Session()
         try:
```

Comment:
**Suggestion:** Consider handling the case where the Bitbucket bearer token is not found in the secrets file. This will improve error handling and make the code more robust. [error handling]